pephub/dependencies.py

```diff
@@ -233,7 +231,7 @@ def get_subsamples(
     project: str,
     tag: Optional[str] = DEFAULT_TAG,
     agent: PEPDatabaseAgent = Depends(get_db),
-) -> Dict[str, Any]:
+) -> Dict[str, Any]:  # type: ignore # type: ignore
```

Comment:
What is that, ignore?

Reply:
I get warnings in VSCode about incorrect type annotations because of `ruff`... and they bother me/prevent me from finding real issues so I suppress them 🥴